PowerValidatedSolutions.psm1

```diff
@@ -51379,46 +51378,134 @@ Function New-AriaNetworksvCenterDataSource {
     #>
 
     Param (
-        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$fqdn,
-        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$username,
-		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$password,
-		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$nickname,
-		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$collectorId,
-		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$enabled
+      [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$fqdn,
+      [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$username,
+      [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$password,
+      [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$nickname,
+      [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$collectorId,
+      [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$enabled
     )
```

Comment:
```suggestion
		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$fqdn,
		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$username,
		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$password,
		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$nickname,
		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$collectorId,
		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$enabled
    )
```

Reply:
Resolved in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/954c74bc51b1fef11dc26b704cc60d52dd1bba80